Wrappers/Python/test/test_algorithms.py

```diff
@@ -259,47 +260,48 @@ def test_PDHG_Denoising(self):
         
         # loader = TestData(data_dir=os.path.join(os.environ['SIRF_INSTALL_PATH'], 'share','ccpi'))
         # loader = TestData(data_dir=os.path.join(sys.prefix, 'share','ccpi'))
-        loader = TestData()
         
-        data = loader.load(TestData.PEPPERS, size=(256,256))
+        data = dataexample.PEPPERS.get(size=(256,256))
         ig = data.geometry
         ag = ig
 
         which_noise = 0
```

Comment:
This confuses me as a unit test. There is a lot of logic applying the noise, but only one type is hardcoded and tested. I guess they should be separate unit tests, or at least iterate through the whole list.

Reply:
let's add the other cases as unit tests later